frontend/src/components/mainPage/codeEditor/subcomponents/Codearea.vue

```diff
@@ -42,6 +43,72 @@ export default {
             this.$emit("pickVariableEvent", variable);
         },
 
+        moveTrackedVariables(changes) {
+            const handleVarTrackerMove = (ch, varObj) => {
```

Comment:
Wydaje mi się, że nie trzeba robić rozgraniczenia względem tego czy ch.size jest dodatnie czy ujemne. Jak dobrze myślę, to można tak samo obsłużyć przypadek dodania i usunięcia fragmentu tekstu. Tym bardziej, że przypadek podmiany kodu może mieć zarówno (w przypadku tej implementacji) size >0, =0 i <0.